vault/dendron.tutorial.user-interface.md

```diff
@@ -0,0 +1,60 @@
+---
+id: gHdxXlNMr1w4xqee0n-Mb
+title: User Interface
+desc: 1. User Interface
+updated: 1639532271917
+created: 1625563862198
+nav_order: 0
+---
+
+Let's do a brief overview on how to navigate the Dendron User Interace (UI), which is called the **[[Workbench|dendron://dendron.dendron-site/dendron.topic.workbench]]**.
+
+> 💡 **TIP:** If you're unfamiliar with VSCode, the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) is used to quickly run commands, including Dendron commands. To bring up the command palette, use `Ctrl+Shift+P` / `Cmd+Shift+P` and start typing the command. If you see instructions telling you to run a command, remember to bring up the command palette and type the command name to run it.
+
+![[dendron://dendron.dendron-site/asset.preview#tutorial-dendron-layout-dark,1:#*]]
+
+> 💡 TIP: VSCode lets you re-position any of these windows, so feel free to rearrange the windows to work best for you! Refer to [VSCode documentation for more information on the VSCode UI](https://code.visualstudio.com/docs/getstarted/userinterface).
+
+#### 1. Editor View
+
+Where you can write and edit your notes. **Notes** in Dendron are [[markdown|dendron://dendron.dendron-site/dendron.topic.markdown]] files (ex. `my.note.md`).
+
+#### 2. Preview View
+
+Shows the rendered Markdown of your currently opened note. If you close this pane, you can bring it back with the `Dendron: Show Preview` command.
+
+>💡 TIP: The preview view is **read-only** and cannot be used to edit notes. All editing must be done in the editor view.
+
+#### 3. File Explorer Pane
+
+This shows the files and folders in your vault. A **[[vault|dendron://dendron.dendron-site/dendron.topic.vaults]]** stores your notes and assets (ex. images, pdfs, etc.).
+
+![[dendron.tutorial.user-interface#^tree-view-vs-file-explorer]]
+
+#### 4. Outline Pane
+
+Shows an outline of the currently opened note by listing the headers for navigation. Jump around areas of larger notes by clicking on the headers in this pane.
+
+#### 5. Calendar View Pane
+
+![[dendron://dendron.dendron-site/dendron._ref.web-ui#^calendar-view-definition]]
+
+![[dendron://dendron.dendron-site/dendron.topic.special-notes#daily-journal:#*]]
```

Comment:
this is a little bit too much. the ui tutorial should just show you what the different pieces of UI do and not go into specific commands. can we replace this with two sentences about what daily journal is?